packages/dev/core/src/Inputs/scene.inputManager.ts

```diff
@@ -978,8 +980,16 @@ export class InputManager {
                     if (eventData.inputIndex === PointerInput.LeftClick) {
                         if (attachDown && deviceSource.getInput(eventData.inputIndex) === 1) {
                             this._onPointerDown(eventData);
+                            this._activeTouchCount++;

```

Comment:
I remember we add the total pointer count in the code ? what is the difference with this one ?

Reply:
Yeah, I completely forgot about that one!  Initially, I was just trying to track just touches but that param should do roughly the same thing.  The only difference is that `_totalPointersPressed` could account for the edge case of one mouse and one touch pointer.  In any case, let me make the change and verify that it works and then I'll update the PR.